ts/parts/PieSeries.ts

```diff
@@ -1051,29 +1051,37 @@ seriesType<Highcharts.PieSeries>(
         drawEmpty: function (this: Highcharts.PieSeries): void {
             var centerX,
                 centerY,
+                start = this.startAngleRad,
+                end = this.endAngleRad,
                 options = this.options;
 
             // Draw auxiliary graph if there're no visible points.
             if (this.total === 0) {
                 centerX = this.center[0];
                 centerY = this.center[1];
 
-                if (!this.graph) { // Auxiliary graph doesn't exist yet.
-                    this.graph = this.chart.renderer.circle(centerX,
-                        centerY, 0)
+                if (!this.graph) {
+                    this.graph = this.chart.renderer
+                        .arc(centerX, centerY, this.center[1] / 2, 0, start, end)
                         .addClass('highcharts-graph')
                         .add(this.group);
                 }
 
-                this.graph.animate({
+                this.graph.attr({
+                    d: Highcharts.SVGRenderer.prototype.symbols.arc(
+                        centerX,
+                        centerY,
+                        this.center[2] / 2,
+                        0, {
+                            start: start,
+                            end: end,
+                            innerR: options.innerSize
```

Comment:
This fails when `innerSize` is percent. Instead, use the computed value `this.center[3] / 2`. Test case: 
```js
Highcharts.chart('container', {
    series: [{
        data: [null, null, null],
        startAngle: -90,
        endAngle: 90,
        type: 'pie',
        innerSize: '50%'
    }]
});
```